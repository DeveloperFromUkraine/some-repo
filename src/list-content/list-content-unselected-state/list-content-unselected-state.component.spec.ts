import { ComponentFixture, TestBed } from '@angular/core/testing';
import { ListContentUnselectedStateComponent } from './index';
import { ComponentTest } from '../../../test/test-bed/component';

describe('List Content Unselected State', () => {
  let fixture: ComponentFixture<ListContentUnselectedStateComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [ListContentUnselectedStateComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(ListContentUnselectedStateComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
