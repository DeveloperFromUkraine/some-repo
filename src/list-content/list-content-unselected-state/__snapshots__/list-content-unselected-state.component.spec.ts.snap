// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContentUnselectedStateComponent should match snapshot 1`] = `
<ign-list-content-unselected-state>
  
</ign-list-content-unselected-state>
`;
